### PR TITLE
RDCC-872: Merging Changes To Master

### PR DESCRIPTION
**Pull request by AdilOoze**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDCC-872

### Change description ###

Merging changes from RDCC-872 into Master

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[✔︎] No
```

### CODE REVIEW

File: charts/rd-user-profile-api/values.preview.template.yaml [hunk 1]
```diff
@@ -9,11 +9,10 @@ java:
     POSTGRES_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
```
Comment (shreedhar-hmcts) [inline charts/rd-user-profile-api/values.preview.template.yaml]: remove aadIdentityName from this file and increase chart version by 1 and then see PR is green

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -1,7 +1,6 @@
 #!groovy
```
Comment (shreedhar-hmcts) [inline Jenkinsfile_CNP]: remove enableDockerBuild() from this file

File: config/owasp/suppressions.xml [hunk 3]
```diff
@@ -368,109 +312,12 @@
     </suppress>
     <suppress>
         <notes><![CDATA[
-       file name: selenium-opera-driver-3.14.0.jar
+       file name: selenium-opera-driver-3.141.59.jar
        ]]></notes>
```
[the remark this answers is not in the document]
Comment (AdilOoze) [reply config/owasp/suppressions.xml]: I will remove this suppression and see what the dependency-report produces.

File: charts/rd-user-profile-api/values.preview.template.yaml (hunk 1 above)
Comment (AdilOoze) [reply charts/rd-user-profile-api/values.preview.template.yaml]: This has been addressed in this commit: https://github.com/hmcts/rd-user-profile-api/pull/252/commits/346362c6450c88462f59bf41431bfe83d48238ac

File: Jenkinsfile_CNP (hunk 2 above)
Comment (AdilOoze) [reply Jenkinsfile_CNP]: This has been addressed in this commit: https://github.com/hmcts/rd-user-profile-api/pull/252/commits/346362c6450c88462f59bf41431bfe83d48238ac

File: config/owasp/suppressions.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (AdilOoze) [reply config/owasp/suppressions.xml]: Dependency Check report did not publish any vulnerabilities so this suppression can remain removed.

**Comment by AdilOoze** [comment]
The Jenkins build failed due to "helm upgrade failed with error has no deployed releases"  - there are no previously deployed releases for this PR. Will be closing this PR and opening a new one. All review comments have been addressed prior to closure.

**Pull request closed without merging**